docs/glossary.md

```diff
@@ -0,0 +1,265 @@
+
+#### Decrediton
+
+A GUI (Graphical User Interface) [Decred wallet](https://github.com/decred/decrediton/) maintained by the core team. 
+
+#### Difficulty
+
+Difficulty is a measure of how difficult it is to mine a new block (i.e. find a hash below a given target). In Decred, the PoW difficulty is calculated from the exponentially weighted average of differences in previous block times.  
+
+#### Distributed ledger
+
+A cryptographically secure ledger, composed of nodes in a network, where each node holds a copy of the same ledger. 
+
+#### Expired ticket
+
+Tickets that reached the end of their window without being called to vote - these can be revoked, but do not grant a reward.
+
+#### Faucet
+
+A mechanism for obtaining free (testnet) tokens.
+
+#### Hash
+
+The output of a cryptographic hashing function that produces a fixed-size and unique hash value from variable-size input. 
+
+#### Hash function
+
+A cryptographic function that produces a fixed-size and unique hash value from variable-size transaction input. Decred uses the [BLAKE-256](https://docs.decred.org/research/blake-256-hash-function/) hashing function. 
+
+#### Hashrate
+
+The number of hashes per second computed by miners on the network.
+
+#### Hybrid PoW/PoS
+
+In Decred, a [hybrid PoW/PoS system](https://docs.decred.org/research/hybrid-design/) is used, whereby blocks mined by PoW miners must be approved by ticket holders, which are pseudorandomly selected from the ticket pool. This provides a check on PoW miners and increases the overall cost of attacking the network. When a block is approved, 60% of the block reward goes to the PoW miner, 30% goes to the holders of  tickets called to vote (approve blocks), and 10% goes to the Decred Treasury to fund the project's development.
+
+#### Immature ticket
+
+Once tickets are mined they are immature for 256 blocks (about 20 hours) and cannot be called to vote until after this period elapses.
+
+#### Inflation
+
+Increase in the [available supply](https://docs.decred.org/advanced/inflation/) of Decred as new DCR is minted into existence through the block reward.
+
+#### Live ticket
+
+Tickets that are waiting to be called.
+
+#### Mainnet
+
+The proper Decred network - a term used to differentiate from testnet.
+
+#### Mempool
+
+Transactions waiting to be mined.
+
+#### Miners
+
+Miners help secure the network via Proof-of-Work (PoW) mining. Miners recieve block rewards and transaction fees for successfully validating blocks. 
+
+#### Mining Pool
+
+A group of miners who share (pool) their computational resources to mine DCR. When a member of a mining pool successfully mines a block, the rewards are split among all pool members in proportion to the hashpower they contribute to the pool. 
+
+#### Missed ticket
+
+Tickets that have been called but did not respond - these can be revoked, but do not grant a reward.
+
+#### Orphan
+
+A valid block which is not built upon and therefore not included in the definitive blockchain.
```

Comment:
There are two different things that "orphan" can refer to.  This is term for orphaned blocks, but it is also possible to have orphaned transactions (transactions with missing inputs -- their "parent" transactions are unknown making them "orphans").

Reply:
This definition is incorrect.  A block that is not built upon is not an orphan.  In the specific context this is referring to, it is a block that is part of a branch that has been abandoned.

Also, to expand on what @jrick said, "orphan" can also refer to a block that builds on top of another block that is missing as well as transactions that were in an orphaned block and have not been included in another one.

So, to summarize, orphans can refer to:

- Blocks that are a member of a branch of the blockchain that has been abandoned
- Blocks that build on an unknown block (the "parent" block is unknown making it an "orphan")
- Transactions with missing inputs (the "parent" transactions are unknown making them "orphans"
- Transactions that were part of a block that has been abandoned and have not been included in another block.  For example, a common case of this is for coinbase transactions and votes which are not allowed to reappear in another block.

Reply:
Have split this into two definitions. Got the orphaned transaction definition from 'Mastering Bitcoin', Andreas Antonopoulos' book, which is available in its entirety on Google Books it appears. I think he wouldn't mind:) 

#### Orphan block

Orphaned blocks are valid blocks which are not included in the definitive blockchain. They can occur naturally when two miners produce blocks at similar times. They can also be caused by an attacker attempting to reverse transactions in a 51% attack. 


#### Orphan transaction

A transaction with inputs referring to a transaction that is not yet known, such as a missing parent.

Reply:
@davecgh , have updated definitions to include the extra info you provided (below). This is more of a developer term than an average user term, and I think developers would want a more detailed definition.

#### Orphan block

Orphaned blocks are valid blocks which are not included in the definitive blockchain. Orphan blocks can occur when they are part of a branch of the blockchain that has been abandoned. This can occur naturally when two miners produce blocks at similar times. Orphan blocks can also be created when they build on an unknown block (i.e. the "parent" block is unknown, making it an "orphan"). 

#### Orphan transaction

A transaction with missing inputs (i.e. the "parent" transaction is unknown, making the transaction an "orphan"). Opphan transactions can also be created when they are part of a block that has been abandoned and have not been included in another block. For example, if a miner creates a valid block with transactions specific to them, such as coinbase transactions and votes, and that block is orphaned, the transactions in that block will become orphaned transactions.

Reply:
Look good.  I see a typo.  s/Opphan/Orphan/

Reply:
Fixed